### ISSUE TITLE
Make it possible to filter debates by type in the API

### ISSUE THREAD

**Issue by drzax** [openaustralia/openaustralia#558]
For example, I'd like to retrieve all debates from a parliamentary session which are debates on a bill.

As far as I can tell this isn't possible and I'll have to parse the XML to retrieve them.

**Comment by equivalentideas** [comment]
Good idea. This could also be one for the They Vote For You API https://theyvoteforyou.org.au/help/data

But that's about divisions, not debates. It does appear though that you can get an array of bill connected to a division, if that's useful.

**Comment by drzax** [comment]
Though it doesn't solve my current problem, that's good to know, thanks.
